mkdocs_static_i18n/plugin.py

```diff
@@ -78,7 +81,31 @@ def on_nav(self, nav, config, files):
         """
         Translate i18n aware navigation to honor the 'nav_translations' option.
         """
-        i18n_nav = self.reconfigure_navigation(nav, config, files)
+
+        # maybe move to another file and don't pass it as parameter?
+        class NavHelper:
+            translated_items: int = 0
+            homepage: Optional[Page] = None
+
+        i18n_nav = self.reconfigure_navigation(nav, config, files, NavHelper)
+
+        # report translated entries
+        if NavHelper.translated_items:
+            log.info(
+                f"Translated {NavHelper.translated_items} navigation element"
+                f"{'s' if NavHelper.translated_items > 1 else ''} to '{self.current_language}'"
+            )
+
+        if hasattr(i18n_nav, "homepage"):
+            i18n_nav.homepage = NavHelper.homepage
+
+            # assure presence of homepage
+            if self.current_language == self.default_language:
+                i18n_nav.homepage = "/"
```

Comment:
homepage should be a `Optional[Page]`, not `str` please

I also wonder about this, why would you set a default to the default language while you could also do the same for all of them?

I'd rather not set a default tbh.

Reply:
I printed the values of `item.url` and my brain 🥶. If it's supposed to be THE homepage, then I need to look at it again, and actually figure it out 😮‍💨.
I wanted to have a default value here, because it isn't being looked up currently and is `None` and there is the warning, which would break strict. 
My docs have the index page inside of a Section, which is generated by the awesome-pages plugin (the structure was "same" before the switch to using that plugin).

Reply:
We are looking for the homepage on the reconfigre L259 (of your code). I'm not sure what you're talking about?

Reply:
Yes it's being searched for on L259, but it's not being found for the default language `/` path in my `docs`.
I have the warning, which would break `strict` mode. 
I wanted to solve it easily, but the idea was wrong as you pointed out, investigating further...

Reply:
So after my investigation it turned out that:
```py
on_nav Page(title=[blank], url='/mkdocs-static-i18n/')
# ...
on_nav None
```
My docs don't have the homepage at the start of `on_nav`, possibly `awesome-pages` blocks it to then assign it during nav generation ;/
Don't know what to do at this moment to fix it, but I shall just commit my changes. It's either something to be done on `awesome-page`'s end or I could just fork it later and remove the warning.
Anyways...